### PR TITLE
#162949234 Track edit history of comments

### PR DESCRIPTION
**Pull request by MbugwaSami**
**What does this PR do?**
- Tracks edit history of comments

**Description of the Task to be completed**
- Track edit history of comments
- Display current and previous comments

**How should this be manually tested?**
- Clone the repository: git clone https://github.com/andela/ah-the-unsullied.git
- cd ah-the-unsullied
- git checkout ft-track-comment-history-162949234
- create .env on the ah-the-unsullied folder and add the following
```export DATABASE_URL='postgres://USER:PASSWORD@HOST:PORT/NAME'
     export SECRET_KEY="<your_secret_key_here>"
     export HOST_PASSWORD="<Gmail_Password>"
     export HOST_USER="<Gmail_Username>"
     export SMTP_HOST='smtp.gmail.com'
     export EMAIL_PORT='587
```
- Run the following commands 
```python3 -m venv venv
     source .env
     python manage.py migrate
     python manage.py runserver
```
- open postman and test the app
   
**Relevant PT story**
#162949220

**screenshots**
- N/A

### CODE REVIEW

**Review by kwanj-k — CHANGES_REQUESTED** [review]
Kindly avoid pip freezing, add the dependencies you added to requirements.txt only

**Comment by sylviawanjiku** [comment]
According to the engineering playbook you should use small letters in your commit message

**Review by sylviawanjiku — COMMENTED** [review]
According to the engineering playbook you should use small letters in your commit message

**Review by sylviawanjiku — CHANGES_REQUESTED** [review]
According to the engineering playbook you should use small letters in your commit message

**Review by kwanj-k — DISMISSED** [review]

**Review by sylviawanjiku — DISMISSED** [review]

File: authors/tests/base_test.py [hunk 1]
```diff
@@ -430,3 +435,17 @@ def authentication_token_3(self, ):
         res = self.get_token_signup_user_3()
         token = res.data['token']
         return token
+
+    def edit_comment(self):
+        slug = self.create_article().data['slug']
+        url = reverse("articles:comment", kwargs={"slug": slug})
+        response = self.client.post(url, self.comment_data, format="json")
+        id = response.data['id']
```
Comment (indungu) [inline authors/tests/base_test.py]: `id` is a reserved keyword in python, please change this variable name to `comment_id`.

**Review by indungu — CHANGES_REQUESTED** [review]
Nice work. Please make the requested change.

**Review by kwanj-k — DISMISSED** [review]

**Review by kelvinwm — APPROVED** [review]

**Review by indungu — APPROVED** [review]

**Pull request merged**